shell/platform/common/client_wrapper/include/flutter/method_channel.h

```diff
@@ -122,12 +133,70 @@ class MethodChannel {
     messenger_->SetMessageHandler(name_, std::move(binary_handler));
   }
 
+  // Adjusts the number of messages that will get buffered when sending messages
+  // to channels that aren't fully set up yet. For example, the engine isn't
+  // running yet or the channel's message handler isn't set up on the Dart side
+  // yet.
+  void Resize(int new_size) { ResizeChannel(messenger_, name_, new_size); }
+
+  // Defines whether the channel should show warning messages when discarding
+  // messages due to overflow.
+  //
+  // When |warns| is false, the channel is expected to overflow and warning
+  // messages will not be shown.
+  void SetWarnsOnOverflow(bool warns) {
+    SetChannelWarnsOnOverflow(messenger_, name_, warns);
+  }
+
  private:
   BinaryMessenger* messenger_;
   std::string name_;
   const MethodCodec<T>* codec_;
 };
 
+namespace {
```

Comment:
Everything here should be in an implementation file since it's not templated.

Reply:
Moving this to an implementation file works nicely. 
Everything is green now, thanks!